scripts/test.sh

```diff
@@ -11,6 +11,9 @@ for f in $( git diff-tree $TRAVIS_COMMIT --name-status -r | grep php | grep -v "
 
 ./scripts/lint.sh
 
+export TERMINUS_CACHE_DIR="$HOME/.terminus/testcache"
+export TERMINUS_TOKENS_DIR="$HOME/.terminus/testcache/tokens"
```

Comment:
Yeah. I was trying to get the thing working first and then I was going to clean up all the hardcoding.

I can try removing the tokens dir bit now that the other issues are fixed. This may have been a vain attempt at fixing my auth issues